mllib/src/test/scala/org/apache/spark/mllib/stat/HypothesisTestSuite.scala

```diff
@@ -153,4 +157,63 @@ class HypothesisTestSuite extends SparkFunSuite with MLlibTestSparkContext {
       Statistics.chiSqTest(sc.parallelize(continuousFeature, 2))
     }
   }
+
+  test("1 sample kolmogorov smirnov test") {
```

Comment:
`kolmogorov smirnov` -> `Kolmogorov-Smirnov`

Could you also add a unit test to compare with R? Here is one example:

https://github.com/apache/spark/blob/master/mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala#L214